source/_templates/installations/manager/configure_indexer_connection.rst

```diff
@@ -17,5 +19,13 @@
       </ssl>
   </indexer>
 
-.. End of include file
+If you have more than one Wazuh indexer node, add all the required ``<host>`` entries. The vulnerability detector module will report to the first available node found and will switch to the next one in case of error.
+
+.. code-block:: xml
 
+      <hosts>
+        <host>https://10.0.0.1:9200</host>
+        <host>https://10.0.0.2:9200</host>
+      </hosts>
+
```

Comment:
```suggestion
Vulnerability detection prioritizes reporting to the first node in the list. It switches to the next node in case it's not available.

```

Reply:
Added!